src/Sylius/Component/Core/Model/ShopBillingDataInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Model;
+
+interface ShopBillingDataInterface
+{
+    public function getId(): ?int;
```

Comment:
I think we shouldn't enforce this method, as it serves no purpose - shop billing data should be always assigned to either a channel, an invoice or a credit memo.